bootstrap/src/main/java/io/jenkins/jenkinsfile/runner/bootstrap/Bootstrap.java

```diff
@@ -118,6 +120,12 @@
     @Option(name = "--cli", usage = "Launch interactive CLI.", forbids = { "-v", "--runWorkspace", "-a", "-ns" })
     public boolean cliOnly;
 
+    @Option(name = "--credentials", usage = "YAML credentials definitions to load")
```

Comment:
Would it make sense to pass SCM and Credentials through the same file? IIUC this file is largely optional, credentials supplied through the main Jenkins.yaml should work as well

Reply:
Lemme think on that.

Reply:
Yeah, I like that better. Update incoming.